pkg/azureclients/armclient/util.go

```diff
@@ -167,9 +168,16 @@ func DoHackRegionalRetryDecorator(c *Client) autorest.SendDecorator {
 					regionalErrStr = regionalError.Error()
 				}
 
-				klog.V(6).Infof("Send.sendRegionalRequest failed to get response from regional host, error: '%s'. Ignoring the result.", regionalErrStr)
+				klog.V(6).Infof("Send.sendRegionalRequest failed to get response from regional host, error: %q. Ignoring the result.", regionalErrStr)
 				return response, rerr
 			}
+			if regionalResponse.StatusCode == 200 && (regionalResponse.ContentLength == -1 || regionalResponse.ContentLength == 0) {
+				contentLengthErrStr := fmt.Sprintf("Send.sendRegionalRequest failed to get response from regional host, error: StatusCode is 200 but ContentLength is %d. Ignoring the result.",
```

Comment:
how about changing the error message simpler, e.g. `fmt.Sprintf("empty response with ContentLength %d and StatusCode %d", regionalResponse.ContentLength, response.StatusCode)`?

Reply:
Updated.